### ISSUE TITLE
Documentation clarification request: loss function

### ISSUE THREAD

**Issue by GenaBitu** [FluxML/Flux.jl#67]
Hi,
by having the first argument of the `train!` function named loss, I thought I needed to pass just the loss function (e. g. `Flux.mse`) - Upon further reading it figured out that this wasn't the case as seen in the mnist example in model-zoo...

I think maybe better naming or a clearer documentation would help prevent this issue for future Flux beginners.

Thank you

**Comment by MikeInnes** [comment]
Agreed that this was confusing – I've tried tweaking the wording a bit to clarify that this is a model loss function. Let me know if it still seems unclear, or if you have any better ideas for wording.

**Comment by GenaBitu** [comment]
Hmm, I'm not sure... Because with `model` I would imagine the model itself (e. g. a `Flux.Chain`). And also now there is `model` in section Training at the top and `loss` in Loss Functions.
I am not sure what wording would be really clear. But if I may suggest, I think adding `Flux.train!` to the example in section Loss functions would make it much clearer what you ought to pass to it.

Anyway, thanks for the response (and Flux of course :) ), hope I helped...

**Comment by MikeInnes** [comment]
Ok, I tried once more to make this a bit clearer. This was definitely helpful!

**Comment by GenaBitu** [comment]
Looks good to me now. Thank you!
